website/src/resources/notifications.json

```diff
@@ -0,0 +1,5 @@
+[  {
+    "message": "Recommended Latest Release",
+    "url": "/docs/",
+    "linktext": "v2.11.0"
```

Comment:
can we add an end date also to this text, which will not display the banner post the end date. for example, we can have a case where we are putting a banner for an event. In case we forget to come and update here, it should automatically hide it from displaying.

Reply:
added the option for giving expiry date.

updated structure

```
{
    "message": "Recommended Latest Release",
    "url": "/docs/",
    "linkText": "v2.11.0",
    "hideAfter": "2021-09-03T09:00"
}